pilot/pkg/networking/core/v1alpha3/httproute.go

```diff
@@ -251,10 +251,19 @@ func (configgen *ConfigGeneratorImpl) buildSidecarOutboundHTTPRouteConfig(env *m
 
 	out := &xdsapi.RouteConfiguration{
 		Name:             routeName,
-		VirtualHosts:     virtualHosts,
 		ValidateClusters: proto.BoolFalse,
 	}
 
+	//BAVERY_TODO: Find a better way to handle these environment variables than parsing each time
+	if enableVHDS, _ := strconv.ParseBool(node.Metadata["ENABLE_DYNAMIC_HOST_CONFIGURATION"]); !enableVHDS {
+		out.VirtualHosts = virtualHosts
+	}
+
+	//BAVERY_TODO: Find a better way to handle these environment variables than parsing each time
+	if useDeltaXDS, _ := strconv.ParseBool(node.Metadata["USE_DELTA_XDS"]); useDeltaXDS {
```

Comment:
I'm not sure we need the delta XDS - at least on the short term. What we really need is the on-demand part.